src/pages/[platform]/build-a-backend/add-aws-services/in-app-messaging/identify-user/index.mdx

```diff
@@ -0,0 +1,90 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Identify a user',
+  description: 'Learn how to segment and target your In-App Messaging campaigns to specific user subsets.',
+  platforms: [
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+To fully harness the potential of In-App Messaging, you must segment and target your In-App Messaging campaigns to specific user subsets. By identifying users with additional information, including their device demographics, location and any attributes of your choosing, you will be able to display intelligent, targeted in-app messages to the right users.
+
+```js
+import { identifyUser } from 'aws-amplify/in-app-messaging';
+
+const identifyUserInput = {
+  userId: '', // E.g. user-id
+  userProfile: {
+    email: '', // E.g. example@service.com
+    name: '', // E.g. name-of-the-user
+    plan: '' // E.g. plan-they-subscribe-to
+    customProperties: {
+      // E.g. hobbies: ['cooking', 'knitting'],
+    },
+    demographic: {
+      appVersion: '',
+      locale: '', // E.g. en_US
+      make: '', // E.g. Apple
+      model: '', // E.g. iPhone
+      modelVersion: '', // E.g. 13
+      platform: '', // E.g. iOS
+      platformVersion: '', // E.g. 15
+      timezone: '' // E.g. Americas/Los_Angeles
+    },
+    location: {
+      city: '', // E.g. Seattle
+      country: '', // E.g. US,
+      postalCode: '', // E.g. 98121
+      region: '', // E.g. WA
+      latitude: 0.0,
+      longitude: 0.0
+    },
+    metrics: {
+      // E.g. logins: 157
+    },
+  },
+};
+
+await identifyUser(identifyUserInput);
+```
```

Comment:
````suggestion
```js
import { identifyUser } from 'aws-amplify/in-app-messaging';

await identifyUser({
  userId: '', // E.g. user-id
  userProfile: {
    email: '', // E.g. example@service.com
    name: '', // E.g. name-of-the-user
    plan: '' // E.g. plan-they-subscribe-to
    customProperties: {
      // E.g. hobbies: ['cooking', 'knitting'],
    },
    demographic: {
      appVersion: '',
      locale: '', // E.g. en_US
      make: '', // E.g. Apple
      model: '', // E.g. iPhone
      modelVersion: '', // E.g. 13
      platform: '', // E.g. iOS
      platformVersion: '', // E.g. 15
      timezone: '' // E.g. Americas/Los_Angeles
    },
    location: {
      city: '', // E.g. Seattle
      country: '', // E.g. US,
      postalCode: '', // E.g. 98121
      region: '', // E.g. WA
      latitude: 0.0,
      longitude: 0.0
    },
    metrics: {
      // E.g. logins: 157
    },
  },
});
```
````

defining this object inside the function argument grants type hints. We could also keep the object and import the type used for the arg